addons/im_livechat/models/chatbot_script_question_answer.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields
+from odoo.osv import expression
+
+
+class ChatbotScriptQuestionAnswer(models.Model):
+    _name = 'im_livechat.chatbot.script_question_answer'
+    _description = 'Chatbot Script Question Answer'
+
+    name = fields.Char(string='Name', required=True)
+    sequence = fields.Integer(string='Sequence')
+    redirect_link = fields.Char('Redirect Link')
+    step_id = fields.Many2one(
+        'im_livechat.chatbot.script_step', string='Script Step', required=True, ondelete='cascade')
+    chatbot_id = fields.Many2one(related='step_id.chatbot_id', store=True)
+
+    def _compute_display_name(self):
+        for record in self:
+            record.display_name = record.name
+
+    def name_get(self):
+        result = []
+        for answer in self:
+            answer_message = answer.step_id.message.replace('\n', ' ').strip()
+            if len(answer_message) < 24:
+                message = answer_message
+            else:
+                message = answer_message[:20] + '...'
+
+            result.append((answer.id, "%s : %s" % (message, answer.name)))
```

Comment:
Do we need a space before the ":" ? Seems we don't do it generally (or maybe language dependent ?).